gbfs.md

```diff
@@ -1264,7 +1323,7 @@ Note that the Android URI and iOS Universal Link URLs don’t necessarily use th
     "language": "en",
     "rental_apps": {
       "android": {
-        "discovery_uri": "com.example.android://",
+        "discovery_uri": "com.example.android://"
```

Comment:
@fchabouis is the comma removal voluntary?

Reply:
thanks @thbar I forgot to cherry-pick a [commit](https://github.com/NABSA/gbfs/pull/374) when fixing the formatting issue